tools/stores_l10n/README.md

```diff
@@ -0,0 +1,101 @@
+
+# Projects configuration
+The entire configuration is stored inside the [Project class](https://github.com/mozilla-l10n/stores_l10n/blob/master/app/classes/Stores/Project.php).
+
+`$products_data` stores all supported products, with the following structure for each of them:
+```PHP
+'fx_android' =>
+    [
+        'channels' => ['beta', 'release'],
+        'name'     => 'Firefox for Android',
+        'store'    => 'google',
+    ],
+```
+
+Relevant information in this array:
+* The ID of the project, used also for API calls, it’s the item’s key.
+* Supported channels.
+* Full name of the product, used in views and navigation.
+* In which store the project lives. Currently it can be `apple` or `google`.
+
+`$supported_locales` includes information on all locales supported for each product and channel. It’s important to note that these lists are used in [Langchecker](/tools/webdashboards/langchecker.md), to determine which locales should see specific files (product pages on mozilla.org, store content).
+
+`$locales_mapping` includes information on mappings between Mozilla’s locale codes, and each store’s internal codes.
+
+`$templates` determines, for each product and channel, which template is used, and which .lang files are associated to a specific part of the template (What’s new, main listing, screenshots). This array is updated every time there is a [new release](/products/appstores/README.md).
+
+In order to add a brand new project, you would need to:
+* Define the new project and supported channels in `$products_data`.
+* Define the list of supported locales for each channel in `$supported_locales`.
+* Define an entry in `$templates`.
+* Create files in [/templates](https://github.com/mozilla-l10n/stores_l10n/tree/master/app/templates) and [/view](https://github.com/mozilla-l10n/stores_l10n/tree/master/app/views) for each supported channel.
+* Track the new .lang files in Langchecker for the appstores repository.
+
+# Template and View Structure
+This is a portion of the template file used for Focus for iOS.
+
+```PHP
+$screenshots = function ($translations) use ($_) {
+    return <<<OUT
+{$_('Automatically block ads<br>& other Web trackers')}
+
+{$_('Browse Faster<br>Web pages may load faster<br>by removing trackers')}
+
+{$_('Before')}
+
+{$_('After')}
+
+{$_('Tracker')}
+
+{$_('From Mozilla<br>A brand you trust')}
+OUT;
+};
+```
+
+Similarly to a Django template, each string is wrapped in a translation function `$_('')` (defined in [app/inc/utilities.php](https://github.com/mozilla-l10n/stores_l10n/blob/master/app/inc/utilities.php#L5)).
+Strings are grouped into a variable, like `$screenshots` in this case: this allows, for example, to determine the overall length of all strings included in a section, to see if it exceeds the [limits imposed by stores](https://github.com/mozilla-l10n/stores_l10n/blob/master/app/inc/constants.php#L15-L22).
```

Comment:
Maybe worth mentioning that if translations exceed the max length, they are considered "not done" until localizers fix them (unless this has changed), and that length errors are not displayed in the webdashboard.